airgun/entities/acs.py

```diff
@@ -0,0 +1,823 @@
+import time
+
+from wait_for import wait_for
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep
+from airgun.navigation import navigator
+from airgun.utils import retry_navigation
+from airgun.views.acs import AddAlternateContentSourceModal
+from airgun.views.acs import AlternateContentSourcesView
+from airgun.views.acs import EditCapsulesModal
+from airgun.views.acs import EditCredentialsModal
+from airgun.views.acs import EditDetailsModal
+from airgun.views.acs import EditProductsModal
+from airgun.views.acs import EditUrlAndSubpathsModal
+from airgun.views.acs import RowDrawer
+
+
+class AcsEntity(BaseEntity):
+    def do_refresh(self):
+        """
+        Refresh the page as it is needed to refresh page to see changes after deleting an ACS
+        """
+
+        self.browser.refresh()
+        self.browser.plugin.ensure_page_safe()
```

Comment:
This is not specific to `AcsEntity` and can be moved elsewhere.  I would suggest moving this to `AirgunBrowserPlugin ` if this needs to be done often or I may update the  `refresh` method with an additional call of the plugin.ensure_page_safe()